gateway-provider-security-pac4j/src/main/java/org/apache/knox/gateway/pac4j/filter/Pac4jDispatcherFilter.java

```diff
@@ -75,12 +77,18 @@
 
   public static final String PAC4J_CALLBACK_PARAMETER = "pac4jCallback";
 
+  public static final String PAC4J_AZURE_AD_CLIENT = "AzureAdClient";
+
+  public static final String URL_PATH_SEPARATOR = "/";
```

Comment:
nit: I would guess that there is already a constant for this from HTTP client or something?

Reply:
Ah, I couldn't find any constants that are exposed.

Reply:
Well you are probably right since I couldn't find any. Surprising :(

https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/constant-values.html